.github/workflows/stale-bot.yaml

```diff
@@ -0,0 +1,22 @@
+name: Close stale issues and PRs
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v8
+        with:
+          stale-issue-message: This issue is stale because it has been open 14 days with no activity.
+          stale-pr-message: This pull request is stale because it has been open 14 days with no activity.
+          close-issue-message: This issue was closed because it has been stalled for 2 days with no activity.
```

Comment:
```suggestion
          close-issue-message: This issue was closed because it has been stalled for 2 days with no activity. You are still welcome to reopen it and continue from where you finished. Best regards Saleor team
```